src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -481,5 +482,18 @@ public static bool IsWorkerDynamicConcurrencyEnabled(this IEnvironment environme
             }
             return false;
         }
+
+        public static HashSet<string> GetLanguageWorkerListToStartInPlaceholder(this IEnvironment environment)
+        {
+            string placeholderList = environment.GetEnvironmentVariableOrDefault(RpcWorkerConstants.FunctionWorkerPlaceholderModeListSettingName, string.Empty);
+            var placeholderRuntimeSet = new HashSet<string>(placeholderList.Split(';'));
```

Comment:
nit: 
```suggestion
            var placeholderRuntimeSet = new HashSet<string>(placeholderList.Split(';', StringSplitOptions.RemoveEmptyEntries).Select(p => p.Trim());
```

or `.Split(';', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries)` in .NET 5.